src/Microsoft.OpenApi/Services/OpenApiReferenceResolver.cs

```diff
@@ -4,7 +4,9 @@
 using System;
```

Comment:
This class should be deleted. We don't need it anymore.

Reply:
We've refactored it and are using it to resolve JSON schema references when visiting a JSON schema instance

Reply:
Defer resolution to future PR.